index.js

```diff
@@ -131,17 +110,21 @@ Happy hacking!
 `);
 };
 
-yargs
-    .option('vanilla',{
-        desc: 'create blank plain JS project',
-        type: 'boolean',
-        default: false
+const { frontend, contract, projectDir } = yargs
+    .usage('$0 <projectDir>', 'Create a new NEAR project')
+    // BUG: does not work; https://github.com/yargs/yargs/issues/1331
+    .example('$0 new-app', 'Create a project called "new-app"')
+    .option('frontend', {
+        desc: 'template to use',
+        choices: ['vanilla', 'react'],
+        default: 'vanilla',
     })
-    .option('rust',{
-        desc: 'use rust for smart contract',
-        type: 'boolean',
-        default: false
+    .option('contract', {
+        desc: 'language for smart contract',
+        choices: ['assemblyscript', 'rust'],
+        default: 'assemblyscript'
```

Comment:
I've renamed this from `asc` to `assemblyscript` everywhere for the purpose of making the `--help` output easier to understand without insider knowledge.